packages/plugins/src/cli/deploy.ts

```diff
@@ -276,20 +214,24 @@ export const deploy = async (
     throw new Error(`Unknown execution mode.`)
   }
 
-  const initCodeWithArgsArray = getInitCodeWithArgsArray(actionInputs)
+  const initCodeWithArgsArray = getInitCodeWithArgsArray(
+    deploymentInfo.accountAccesses
+  )
   const configArtifacts = await getConfigArtifacts(initCodeWithArgsArray)
 
   const isSystemDeployed = await checkSystemDeployed(provider)
   const parsedConfig = makeParsedConfig(
     deploymentInfo,
-    actionInputs,
-    gasEstimates,
     isSystemDeployed,
     configArtifacts,
-    dryRunFile.libraries,
-    dryRunPath
+    [] // We don't currently support linked libraries.
```

Comment:
We don't have the `libraries` available to us anymore since we don't use the broadcast file, which is why I hardcoded an empty array here